src/LTO.ts

```diff
@@ -56,62 +67,120 @@ export default class LTO {
 	/**
      * Create an account.
      */
-	public account(
-		address?: string,
-		publicKey?: string,
-		privateKey?: string,
-		keyType = "ed25519",
-		seed?: string,
-		nonce = 0
-	): Account {
-		const factory = this.accountFactories[keyType];
-		const account =
-            seed ? factory.createFromSeed(seed, nonce) :
-                privateKey ? factory.createFromPrivateKey(privateKey) :
-            		publicKey ? factory.createFromPublicKey(publicKey) :
-            			factory.create();
-
-		return LTO.guardAccount(account, address, publicKey, privateKey);
+	public account(settings: IAccountIn = {}): Account {
+		const factory = this.accountFactories[settings.keyType ?? "ed25519"];
+		let account: Account;
+
+		if (settings.seed) {
+			const seed = settings.seedPassword
+				? crypto.decryptSeed(settings.seed, settings.seedPassword, SEED_ENCRYPTION_ROUNDS)
+				: settings.seed;
+			account = factory.createFromSeed(seed, settings.nonce ?? 0);
+		} else if (settings.privateKey) {
+			account = factory.createFromPrivateKey(settings.privateKey);
+		} else if (settings.publicKey) {
+			account = factory.createFromPublicKey(settings.publicKey);
+		} else {
+			account = factory.create();
+		}
+
+		return LTO.guardAccount(account, settings.address, settings.publicKey, settings.privateKey);
 	}
 
 	/**
-     * Encrypt seed phrase
+     * Check if the address is valid for the current network.
      */
-	public encryptSeedPhrase(seedPhrase: string, password: string, encryptionRounds = 5000): string {
-		if (password && password.length < 8)
-			console.warn("Your password may be too weak");
+	public isValidAddress(address: string): boolean {
+		return crypto.isValidAddress(address, this.networkByte.charCodeAt(0));
+	}
 
-		if (encryptionRounds < 1000)
-			console.warn("Encryption rounds may be too few");
 
-		if (seedPhrase.length < DEFAULT_CONFIG.minimumSeedLength)
-			throw new Error("The seed phrase you are trying to encrypt is too short");
+	/**
+	 * Transfer LTO from account to recipient.
+	 * Amount is number of LTO * 10^8.
+	 */
+	public transfer(sender: Account, recipient: string, amount: number, attachment: Uint8Array|string = ""): Promise<Transfer> {
```

Comment:
For the transactions, what if we want to sponsor it before broadcasting? How would we use it?